SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
+
+The bzz handshake is the first message to be exchanged between peers
+after the p2p transport protocol has been successfully negotiated. After
+the handshake each peer should remember the following data about the
+other:
+
+-   Swarm Overlay Address of the peer
+
+-   Whether the peer is a *light* or *full* node
```

Comment:
Missing definition

Reply:
So I am thinking maybe we should hold off this SWIP until the Adaptive Node capabilities protocol SWIP is accepted and merged. Defining full and light in the current implementation is arbitrary at best. With the Capabilities they are both defined as transitional placeholders for more fine-grained conditionals in the future.